### PR TITLE
WIP Building and pushing releases to download.delta.chat on tagging

### PR DESCRIPTION
**Pull request by missytake**
draft #1136

### CODE REVIEW

**Comment by missytake** [comment]
Nice, successfull build job :) download here for testing: https://download.delta.chat/desktop/DeltaChat-1186.AppImage

**Comment by missytake** [comment]
But still the beta.6 core, so it doesn't fix #1182 for me :( (well that wasn't the goal for this PR)

@Jikstra is this what you need? (don't merge before removing the "build on PR" line!)

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      run: |
+        sudo dpkg --add-architecture i386
+        sudo apt update
+        sudo apt install --yes wine-stable wine32
+    - uses: actions/checkout@v1
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+    - name: change build name and Product Name to development id
```
Comment (Simon-Laux) [inline .github/workflows/build-release.yml]: don't need that one for release builds

Comment (missytake) [reply .github/workflows/build-release.yml]: Yeah, just realised and removed it ;)

**Comment by Simon-Laux** [comment]
also  the step for portable windows build is missing

**Comment by Simon-Laux** [comment]
also don't do the mac build and appx build here, those require to be run on a pysical mac computer in our controll and a windows 10 machine

**Comment by missytake** [comment]
Okay :) will look into this later

**Pull request closed without merging**